_layouts/handbook-content--g-devdocs-style.html

```diff
@@ -0,0 +1,126 @@
+---
+layout: default
+---
+
+<main role="main" class="devsite-main-content">
+  <div class="devsite-content">
+    <article class="devsite-article">
+      {%- if page.editable != false -%}
+      <div class="improve right">
+        <a href="https://github.com/openinf/.github/edit/HEAD/{{ page.path }}"
+          >Edit</a
+        >
+      </div>
+      {%- endif -%}
+      <h1 class="devsite-page-title">{{ page.title }}</h1>
+      <div class="devsite-article-body clearfix">
+        <aside id="key-point">
+          <p><strong>Key Point:</strong> {{ page.key_point }}</p>
+        </aside>
```

Comment:
```suggestion
        <aside id="key-point">
          <p><strong>Key Point:</strong> {{ page.key_point }}</p>
        </aside>
```

should be extrapolated

Reply:
ok, let's do this now that i remember what i was talking about!